cmd/thanos/tools.go

```diff
@@ -36,29 +38,41 @@ func registerCheckRules(app extkingpin.AppClause) {
 
 func checkRulesFiles(logger log.Logger, files *[]string) error {
 	var failed errutil.MultiError
+	errhandle := func(e error) {
+		level.Error(logger).Log("result", "FAILED", "error", e)
+		level.Info(logger).Log()
+		failed.Add(e)
+	}
 
 	for _, fn := range *files {
 		level.Info(logger).Log("msg", "checking", "filename", fn)
-		f, err := os.Open(fn)
+		matches, err := filepath.Glob(fn)
 		if err != nil {
-			level.Error(logger).Log("result", "FAILED", "error", err)
-			level.Info(logger).Log()
-			failed.Add(err)
+			errhandle(err)
+			continue
+		}
+		if matches == nil {
+			err = errors.New("matching file not found")
+			errhandle(err)
```

Comment:
We can join these two together

Reply:
> We can join these two together

Like single error as `Invalid patter or matching file not found` ?

Reply:
My idea was that we could simply write:
```
errhandle(errors.New("matching file not found"))
```
:smile: